<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmProds</class>
 <widget class="QMainWindow" name="FrmProds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de manutenção de banco de dados</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #013C31;
</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>461</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FE7411;
border-radius: 15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>391</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff</string>
     </property>
     <property name="text">
      <string>Sistema de manutenção de banco de dados</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff</string>
     </property>
     <property name="text">
      <string>DaRoça</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff</string>
     </property>
     <property name="text">
      <string>Adicione, edite, remova e veja</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff</string>
     </property>
     <property name="text">
      <string>os produtos DaRoça com</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff</string>
     </property>
     <property name="text">
      <string>facilidade e rapidez</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FE7411;
color: #fff;</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Adicionar"/>
   <addaction name="action_Editar"/>
   <addaction name="action_Remover"/>
   <addaction name="action_Lista"/>
   <addaction name="separator"/>
   <addaction name="action_Sair"/>
  </widget>
  <action name="action_Adicionar">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/prod/imgsDaRocaBD/adicionar.png</normaloff>:/prod/imgsDaRocaBD/adicionar.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Adicionar</string>
   </property>
   <property name="toolTip">
    <string>Incluir novo produto</string>
   </property>
  </action>
  <action name="action_Editar">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/prod/imgsDaRocaBD/editar.png</normaloff>:/prod/imgsDaRocaBD/editar.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Editar</string>
   </property>
   <property name="toolTip">
    <string>Alterar os dados de um produto existente</string>
   </property>
  </action>
  <action name="action_Lista">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/prod/imgsDaRocaBD/listar.png</normaloff>:/prod/imgsDaRocaBD/listar.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Lista</string>
   </property>
   <property name="toolTip">
    <string>Apresenta uma lista com todos os produtos</string>
   </property>
  </action>
  <action name="action_Remover">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/prod/imgsDaRocaBD/remover.png</normaloff>:/prod/imgsDaRocaBD/remover.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Remover</string>
   </property>
   <property name="toolTip">
    <string>Excluir um produto existente</string>
   </property>
  </action>
  <action name="action_Sair">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/prod/imgsDaRocaBD/sair.png</normaloff>:/prod/imgsDaRocaBD/sair.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Sair</string>
   </property>
   <property name="toolTip">
    <string>Fecha a conexão</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
